nalgebra-glm/src/ext/matrix_clip_space.rs

```diff
@@ -691,10 +687,110 @@ pub fn perspective_zo<N: RealField>(aspect: N, fovy: N, near: N, far: N) -> TMat
     perspective_rh_zo(aspect, fovy, near, far)
 }
 
+/// Build infinite right-handed perspective projection matrix with [-1,1] depth range.
+///
+/// # Parameters
+///
+/// * `fovy` - Field of view, in radians
+/// * `aspect` - Ratio of viewport width to height (width/height)
+/// * `near` - Distance from the viewer to the near clipping plane.
```

Comment:
Could these (and similar below) be listed in the order they're passed to the function?